### DISCUSSION TITLE
How to do collapsible with lazy load

### DISCUSSION THREAD

**Discussion by novaknole** [docsifyjs/docsify#2203] [General]
I have been writing some docs for my own use case. The library is amazing. Though I need ideas.

Some pages are quite long, so I got tired of scrolling to get to the bottom of the page. You might say that I can create other pages and separate them, but I don't want it. I need to have it on the same page.

I got 2 things I want to find a way about: 
* The page is big, so it's slow(kind of lagging)
* Since page is long, I end up scrolling quite some time to get to the bottom.

I would love to have the following: some collapsibles such as you go to the page, and you see 15 collapsibles and clicking on each of them shows the contet later.  I couldn't find anything through docsify, so I did it with my own html/css and some javascript and I solved the problem about "scrolling too long". But it's ofc still slow.

Isn't there some way that I got collapsibles, though, whatever is hidden behind it is not loaded on the page load and only loaded once clicked ? This way, performance will be much better. Thoughts ?

**Comment by novaknole** [comment]
No one ?

**Comment by jhildenbiddle** [comment]
Closing in favor of #2206.
